hotel/web-application/src/main/java/com/andersenhotels/presenter/Presenter.java

```diff
@@ -0,0 +1,97 @@
+package com.andersenhotels.presenter;
+
+import com.andersenhotels.model.entities.Apartment;
+import com.andersenhotels.model.entities.Hotel;
+import com.andersenhotels.model.exceptions.*;
+import com.andersenhotels.model.service.logic.HotelServiceWrapper;
+import lombok.Getter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class Presenter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Presenter.class);
```

Comment:
yiu can use logging-starter  + lombok
1) remove logger depencencies
2) add logger starter
3) use annotation Slf4j on the class level